contrib/docker/.env

```diff
@@ -1 +1,2 @@
 COMPOSE_PROJECT_NAME=superset
+SUPERSET_PORT=8088
```

Comment:
@oliviermichaelis I don't think it is a good idea to put the env var here: it should reside inside the docker-compose file, so that it is visible to users.
The reason there is an `.env` file is only because it is the only way to configure a project name in a persistent way.

Reply:
@victornoel I totally agree with you!